safemysql.class.php

```diff
@@ -506,7 +526,19 @@ protected function prepareQuery($args)
 				continue;
 			}
 
-			$value = array_shift($args);
+			if ( !$namedPlaceholders || strlen($part) == 2 )
+			{
+				$value = array_shift($values);
+			} else {
+				$parameter = substr($part, 3);
+				if (!isset($args[$parameter]))
```

Comment:
`isset()` don't allow using `null` values in named parameter. I propose to change to `if (!array_key_exists($parameter, $args))`